docs/advanced_reference/commands/autonomy_deploy.md

```diff
@@ -114,6 +114,20 @@ autonomy deploy build keys.json -ltm
 
 Builds a service deployment using the keys stored in the file `keys.json` and applying environment variables to the service configuration file. The deployment will be generated by default for as many agents as keys are stored in `keys.json`. By default, the command searches for the file `keys.json`, if no file name is provided.
 
+If you are manually defining `keys.json` make sure to include `ledger` property for a key object. This will be used when writing key files to the deployment environment. If you don't specify the property `ethereum` will be used as default ledger identifier, so if you're running an agent against a different ledger (eg. `solana`) you'll have to specify this property manually like this
+
+```json
+[
+    {
+        "address": "Solanaaddress",
+        "private_key": "Solanaprivatekey",
+        "ledger": "solana"
+    }
+]
+```
+
+If the keys are generated using `autonomy generate-key` command the `ledger` property will be included automatically.
```

Comment:
This will address the solana service issue @dvilelaf

Reply:
cc @jmoreira-valory

Reply:
Does this means that the user has to
1. call generate-keys ethereum --> file 1
2. call generate-keys solana --> file 2
3. manually merge file1 and file2

?

Reply:
You only require 1 keys.json file containing solana keys.

Reply:
When building a deployment you only need a keys.json file containing the keys for the default/primary ledger.